CHANGES.rst

```diff
@@ -6,6 +6,8 @@ For changes before version 3.0, see ``HISTORY.rst``.
 7.3 (unreleased)
 ----------------
 
+- Drop support for Python 3.8.
+
```

Comment:
Hmm, no preliminary support / testing for 3.14?

Reply:
AccessControl depends on RestrictedPython, where the Python support is explicitly limited and no Python 3.14-compatible release is available. Those are usually done closer to the first Python RC when there's no chance for breaking changes.